next.clayui.com/content/docs/css/components/labels.md

```diff
@@ -0,0 +1,412 @@
+---
+title: "Labels"
+description: "Labels categorize information, providing quick recognition."
+---
+
+<div class="nav-toc">
+
+- [Label Variations](#label-variations)
+- [Sizes](#sizes)
+- [Variations](#variations)
+	- [Simple Label](#simple-label)
+	- [Label Dismissible](#label-dismissible)
+	- [Label with Link](#label-with-link)
+	- [Interactive Label](#interactive-label)
+	- [Interactive Label Examples](#interactive-label-examples)
+	- [Labels as an Anchor Tag](#labels-as-an-anchor-tag)
+
+</div>
+
+<div class="clay-site-alert alert alert-info">
+	Check the <a href="https://liferay.design/lexicon">Lexicon</a> <a href="https://liferay.design/lexicon/core-components/labels/">Labels Pattern</a> for a more in-depth look at the motivations and proper usage of this component.
+</div>
+
+<div class="clay-site-alert alert alert-warning">
+	See the implementation of the <a href="/docs/components/labels.html">Labels component in React</a> following the Lexicon <a href="https://liferay.design/lexicon/core-components/labels/">Labels Pattern</a>.
+</div>
+
+### Label Variations
+
+<div class="sheet-example">
+	<span class="label label-primary"><span class="label-item label-item-expand">Primary</span></span>
+	<span class="label label-secondary"><span class="label-item label-item-expand">Secondary</span></span>
+	<span class="label label-info"><span class="label-item label-item-expand">Info</span></span>
+	<span class="label label-warning"><span class="label-item label-item-expand">Warning</span></span>
+	<span class="label label-danger"><span class="label-item label-item-expand">Danger</span></span>
+	<span class="label label-success"><span class="label-item label-item-expand">Success</span></span>
+	<span class="label label-light"><span class="label-item label-item-expand">Light</span></span>
+	<span class="label label-dark"><span class="label-item label-item-expand">Dark</span></span>
+</div>
+
+```html
+<span class="label label-primary"><span class="label-item label-item-expand">Primary</span></span>
+<span class="label label-secondary"><span class="label-item label-item-expand">Secondary</span></span>
+<span class="label label-info"><span class="label-item label-item-expand">Info</span></span>
+<span class="label label-warning"><span class="label-item label-item-expand">Warning</span></span>
+<span class="label label-danger"><span class="label-item label-item-expand">Danger</span></span>
+<span class="label label-success"><span class="label-item label-item-expand">Success</span></span>
+<span class="label label-light"><span class="label-item label-item-expand">Light</span></span>
+<span class="label label-dark"><span class="label-item label-item-expand">Dark</span></span>
+```
+
+### Sizes
+
+> Use `label-lg` to make the label larger, or use the mixin label-size($sassMap) to create a custom sized label:
```

Comment:
We no longer need to add these texts in blockquotes. The same for the others on this page.